package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview",
     "migrate": "nbb -m migrate-sparkeats place placeImage review reviewImage",
     "postinstall": "husky install",
+    "lint": "eslint .",
```

Comment:
Two requests here @JoelLau:

1. Run this in `src` only.
2. Rename to `lint:ts` to follow the pattern of `lint:css` below (although I realize it should be `lint:scss` now!).